Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3595,3 +3595,29 @@ def batch(iterable, batch_size=1):
 
 class DemistoException(Exception):
     pass
+
+
+class IncidentMirror(object):
```

Comment:
Do we really need this class? It looks to me like a simply function is sufficient. @yaakovi @anara123 what do you think?

Reply:
We don't really need that, but we can make the return_results work with this object as well, which will make things less confusing for developers